pkgs/development/libraries/folly/macos-10.11.patch

```diff
@@ -0,0 +1,88 @@
+--- a/folly/IndexedMemPool.h
```

Comment:
I can build folly without this patch. Is this for uses in the code built with clang 5?